freqtrade/exchange/kraken.py

```diff
@@ -124,3 +127,42 @@ def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dic
                 f'Could not place sell order due to {e.__class__.__name__}. Message: {e}') from e
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
+
+    def fill_leverage_brackets(self):
+        """
+            Assigns property _leverage_brackets to a dictionary of information about the leverage
+            allowed on each pair
+        """
+        # TODO-lev: Not sure if this works correctly for futures
+        leverages = {}
+        for pair, market in self._api.load_markets().items():
+            info = market['info']
+            leverage_buy = info['leverage_buy']
+            leverage_sell = info['leverage_sell']
+            if len(info['leverage_buy']) > 0 or len(info['leverage_sell']) > 0:
+                if leverage_buy != leverage_sell:
+                    print(f"\033[91m The buy leverage != the sell leverage for {pair}."
+                          "please let freqtrade know because this has never happened before"
+                          )
```

Comment:
Use logging (in this case, most likely warning) - not print to show messages.

Also, absolutely avoid color coding. While it may work on your system, it's not universally applicable and will fail / display wrongly depending on the terminal (powershell/cmd, i'm looking at you specifically) and terminal settings.